src/error.js

```diff
@@ -1,12 +1,15 @@
 // Error
 // -----
 
+import _       from 'underscore';
+import helpers from './helpers';
+
 var errorProps = ['description', 'fileName', 'lineNumber', 'name', 'message', 'number'];
 
-Marionette.Error = Marionette.extend.call(Error, {
-  urlRoot: 'http://marionettejs.com/docs/v' + Marionette.VERSION + '/',
+var MarionetteError = helpers.extend.call(Error, {
```

Comment:
I renamed this to MarionetteError to avoid confusion. Comapre to having `var Error = ...;`.